apparmor.d/groups/xfce/xfdesktop

```diff
@@ -9,15 +9,25 @@ include <tunables/global>
 @{exec_path} = @{bin}/xfdesktop
 profile xfdesktop @{exec_path} {
   include <abstractions/base>
+  include <abstractions/desktop>
```

Comment:
`<abstractions/desktop>` is not needed here as there is already `<abstractions/xfce>`